docs/docs/adding-page-transitions-with-plugin-page-transitions.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Adding Page Transitions with page-transitions
```

Comment:
It would help to use the full plugin name here: `Adding Page Transitions with gatsby-plugin-page-transitions`